game/tournament.py

```diff
@@ -81,13 +81,12 @@ def update_players(self):
         self.running = True
         print(f"Tournament started, updating every {self.update_interval} seconds")
         while self.running:
-            t = time.time()
-            if t - last_update < self.update_interval:
-                continue
-            last_update = t
+            time.sleep(self.update_interval)
```

Comment:
Sleep instead of looping until the update interval is met.

Sleeping stops the thread, allowing the other threads to be better scheduled (especially because of python's global interpreter lock + less load in general).

On my machine, this *greatly* reduces the app's processor usage.